keps/sig-storage/20180515-svcacct-token-volumes.md

```diff
@@ -0,0 +1,180 @@
+---
+title: Service Account Token Volumes
+authors:
+  - "@smarterclayton"
+  - "@liggitt"
+  - "@mikedanese"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-auth
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: "@zshihang"
+creation-date: 2018-05-15
+last-updated: 2020-03-04
+status: implemented
+see-also:
+  - "https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/svcacct-token-volume-source.md"
```

Comment:
Add a commit hash to the URL:

https://github.com/kubernetes/community/blob/e3a6b8172fec9506b15520549e52be683e30cbfb/contributors/design-proposals/storage/svcacct-token-volume-source.md